rotkehlchen/chain/ethereum/modules/balancer/types.py

```diff
@@ -299,7 +299,7 @@ def serialize(self) -> Dict[str, Any]:
 
         return {
             'pool_address': self.pool_address_token.evm_address,
-            'pool_tokens': tokens_and_weights.sort(key=lambda x: x['token']),
+            'pool_tokens': tokens_and_weights,
```

Comment:
Why was the `.sort(key=lambda x: x['token'])` removed? The order no longer matters or am I missing something?

Reply:
Yes, the order didn't matter as can be seen here https://github.com/rotki/rotki/blob/bugfixes/rotkehlchen/chain/ethereum/modules/balancer/types.py#L309 . It was a change I introduced by mistake when fixing the balancer protocol at the moment of introducing the globaldb migration

Reply:
Okayyyy. it looks good to me then